try {
	dosomething()
}
catch(Exception e) {
	logger.log(Level.WARNING, "Something went wrong");
}
